### PR TITLE
Add details for "Battery Strategy" and "Battery Smart Control" sensors

### PR DESCRIPTION
**Pull request by mzurhorst**
This PR ammends the documentation with the info requested in: https://github.com/home-assistant/home-assistant.io/issues/28020

Github user "stegm" provided a link to the Kostal Plenticore Plus documentation in his comment.  (https://github.com/home-assistant/home-assistant.io/issues/28020#issuecomment-1763371033) 

Background information for the sensors "Battery Strategy" and "Battery Smart Contol" has been added accoding to the Kostal documentation.

Link to source document from Kostal server:  https://www.kostal-solar-electric.com/de-de/download/download/-/media/document-library-folder---kse/2020/12/15/13/28/plenticore-plus_ba_en.pdf

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by stegm — APPROVED** [review]
Thx

File: source/_integrations/kostal_plenticore.markdown [hunk 1]
```diff
@@ -128,6 +128,17 @@ The following sensors are available in the library:
 Setting values change less often, therefore these sensors are only polled every 5 minutes.
 </div>
 
+#### Battery Strategy
+This sensor is on by default, which maps to the "Automatically" mode in the Kostal Plenticore Plus documentation. This mode is recommended for regions with little snowfall.
+
+Turning this sensor off maps to the "Automatically economical" mode. Consequently, the inverter controls the batter charging automatically, but switches the battery off when there is not enough PV energy to charge the battery for longer periods. This mode is recommended for regions with a lot of snowfall.
```
Comment (frenck) [inline source/_integrations/kostal_plenticore.markdown]:
```suggestion

This sensor is on by default, which maps to the "Automatically" mode in the Kostal Plenticore Plus documentation. This mode is recommended for regions with little snowfall.

Turning this sensor off maps to the "Automatically economical" mode. Consequently, the inverter controls the battery charging automatically but switches the battery off when there is insufficient PV energy to charge the battery for longer periods. This mode is recommended for regions with a lot of snowfall.
```

File: source/_integrations/kostal_plenticore.markdown [hunk 2]
```diff
@@ -128,6 +128,17 @@ The following sensors are available in the library:
...
+
+#### Battery Smart Control
+The Battery Smart Control sensor appears as a select field labelled "Battery Charging / Usage Mode" with three options:
+- None: the battery is loaded immediately when there is PV energy spare
+- Battery:SmartBatteryControl:Enable: the battery loading optimizes grid feed-in and battery loading. This setting is recommended when the grid feed-in is limited to e.g. 70% of the Plenticore Plus peak power.
+- Battery:TimeControl:Enable: battery charging/discharging can be configured flexibly at different times (tariff periods). Detailed settings have to be done on the webfrontend of the Kostal Plenticore Plus inverter. This option activates the time-controlled battery usage mode.
```
Comment (frenck) [inline source/_integrations/kostal_plenticore.markdown]:
```suggestion
The Battery Smart Control sensor appears as a select field labeled "Battery Charging / Usage Mode" with three options:

- **None**: the battery is loaded immediately when there is PV energy spare.
- **Battery:SmartBatteryControl:Enable**: the battery loading optimizes grid feed-in and battery loading. This setting is recommended when the grid feed-in is limited to, for example, 70% of the Plenticore Plus peak power.
- **Battery:TimeControl:Enable**: battery charging/discharging can be configured flexibly at different times (tariff periods). Detailed settings must be done on the web frontend of the Kostal Plenticore Plus inverter. This option activates the time-controlled battery usage mode.
```

File: source/_integrations/kostal_plenticore.markdown [hunk 3]
```diff
@@ -128,6 +128,19 @@ The following sensors are available in the library:
...
 </div>
 
+#### Battery Strategy
+
+This sensor is on by default, which maps to the "Automatically" mode in the Kostal Plenticore Plus documentation. This mode is recommended for regions with little snowfall.
+
+Turning this sensor off maps to the "Automatically economical" mode. Consequently, the inverter controls the battery charging automatically but switches the battery off when there is insufficient PV energy to charge the battery for longer periods. This mode is recommended for regions with a lot of snowfall.
+
+#### Battery Smart Control
+The Battery Smart Control sensor appears as a select field labeled "Battery Charging / Usage Mode" with three options:
```
Comment (frenck) [inline source/_integrations/kostal_plenticore.markdown]:
```suggestion
#### Battery Smart Control

The Battery Smart Control sensor appears as a select field labeled "Battery Charging / Usage Mode" with three options:
```

**Review by frenck — APPROVED** [review]
Thanks, @mzurhorst 👍 

../Frenck

**Pull request merged**